src/commands/sites/sites-create-template.js

```diff
@@ -0,0 +1,251 @@
+// @ts-check
+
+const slugify = require('@sindresorhus/slugify')
+const inquirer = require('inquirer')
+const pick = require('lodash/pick')
+const sample = require('lodash/sample')
+const fetch = require('node-fetch')
+const prettyjson = require('prettyjson')
+const { v4: uuidv4 } = require('uuid')
+
+const { chalk, error, log, logJson, track, warn } = require('../../utils')
+const { getGitHubToken } = require('../../utils/gh-auth')
+
+const SITE_NAME_SUGGESTION_SUFFIX_LENGTH = 5
+
+let ghToken
+
+// Templates are hardcoded for now
+const templates = [
+  {
+    name: 'Next.js Blog theme',
+    sourceCodeUrl: 'https://github.com/netlify-templates/nextjs-blog-theme',
+    slug: 'netlify-templates/nextjs-blog-theme',
+  },
+  {
+    name: 'Next.js Starter',
+    sourceCodeUrl: 'https://github.com/netlify-templates/next-netlify-starter',
```

Comment:
Do we maybe want to auto fetch the `netlify-templates` by doing a fetch to github? Instead of hardcoding them. This would mean we would need to create a new release for adding a new template and if we would autofetch them it could be just add a new repo inside this org.

Reply:
I didn't wanna do it for now because the UI doesn't do it and at the moment, there's a lot of templates in the `netlify-templates` org that we don't actually want people to use. We've been archiving most of them cause we're kinda starting from scratch.

Reply:
After talking with the team, I actually updated the code to fetch directly from the Github repo 🙂